### PR TITLE
replace words in "replace/contrib"

### PR DESCRIPTION
**Pull request by chipstar**
The replaced words are as follows.

Words:

bitcoin -> xpchain
Bitcoin -> XPChain
BITCOIN -> XPCHAIN
Websites:

bitcoin.org -> xpchain.io
bitcoincore.org -> xpchaincore.org
Repositories:

bitcoin/bitcoin -> xpchain/xpchain

### CODE REVIEW

File: contrib/init/README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-OpenRC:  bitcoind.openrc
-         bitcoind.openrcconf
-CentOS:  bitcoind.init
-OS X:    org.bitcoin.bitcoind.plist
+SystemD: xpchaind.service
+Upstart: xpchaind.conf
+OpenRC:  xpchaind.openrc
+         xpchaind.openrcconf
+CentOS:  xpchaind.init
+OS X:    org.xpchain.xpchaind.plist
```
Comment (foo87b) [inline contrib/init/README.md]: is this ok? @Naruto

File: contrib/devtools/gen-manpages.sh [hunk 2]
```diff
@@ -4,23 +4,23 @@ TOPDIR=${TOPDIR:-$(git rev-parse --show-toplevel)}
...
+XPCHAINCLI=${XPCHAINCLI:-$SRCDIR/xpchain-cli}
+XPCHAINTX=${XPCHAINTX:-$SRCDIR/xpchain-tx}
+XPCHAINQT=${XPCHAINQT:-$SRCDIR/qt/xpchain-qt}
 
-[ ! -x $BITCOIND ] && echo "$BITCOIND not found or not executable." && exit 1
+[ ! -x $XPCHAIND ] && echo "$XPCHAIND not found or not executable." && exit 1
 
 # The autodetected version git tag can screw up manpage output a little bit
-BTCVER=($($BITCOINCLI --version | head -n1 | awk -F'[ -]' '{ print $6, $7 }'))
+BTCVER=($($XPCHAINCLI --version | head -n1 | awk -F'[ -]' '{ print $6, $7 }'))
```
Comment (foo87b) [inline contrib/devtools/gen-manpages.sh]: BTC is bitcoin symbol

File: contrib/README.md [hunk 3]
```diff
@@ -22,29 +22,29 @@ Build Tools and Keys
...
 
 ### [Debian](/contrib/debian) ###
-Contains files used to package bitcoind/bitcoin-qt
-for Debian-based Linux systems. If you compile bitcoind/bitcoin-qt yourself, there are some useful files here.
+Contains files used to package xpchaind/xpchain-qt
+for Debian-based Linux systems. If you compile xpchaind/xpchain-qt yourself, there are some useful files here.
 
 ### [Gitian-descriptors](/contrib/gitian-descriptors) ###
-Files used during the gitian build process. For more information about gitian, see the [the Bitcoin Core documentation repository](https://github.com/bitcoin-core/docs).
+Files used during the gitian build process. For more information about gitian, see the [the XPChain Core documentation repository](https://github.com/bitcoin-core/docs).
```
Comment (Naruto) [inline contrib/README.md]: URL is bitcoin url yet.

File: contrib/devtools/update-translations.py [hunk 4]
```diff
@@ -26,7 +26,7 @@
 # Name of transifex tool
 TX = 'tx'
 # Name of source language file
-SOURCE_LANG = 'bitcoin_en.ts'
+SOURCE_LANG = 'xpchain_en.ts'
```
Comment (Naruto) [inline contrib/devtools/update-translations.py]: Qt .ts file name isn't replace to xpchain name yet.

File: contrib/testgen/base58.py [hunk 5]
```diff
@@ -1,8 +1,8 @@
-# Copyright (c) 2012-2017 The Bitcoin Core developers
```
Comment (Naruto) [inline contrib/testgen/base58.py]: DO NOT CHANGE Copyright.

Comment (chipstar) [reply contrib/testgen/base58.py]: i fixed it.

File: contrib/devtools/update-translations.py (hunk 4 above)
Comment (chipstar) [reply contrib/devtools/update-translations.py]: fixed

File: contrib/devtools/gen-manpages.sh (hunk 2 above)
Comment (chipstar) [reply contrib/devtools/gen-manpages.sh]: changed to XPC

**Comment by foo87b** [comment]
LGTM.
Since it is necessary to investigate the domain, temporarily block merging.

**Pull request closed without merging**